fastai/sgdr.py

```diff
@@ -490,6 +542,9 @@ def update(self):
 
 
 class OptimScheduler(LossRecorder):
+    '''
```

Comment:
Could you put all the one-liner docs on a single line? (i.e. put the quotes on the same line)